scripts/format_source.py

```diff
@@ -0,0 +1,17 @@
+import json
+from typing import List, cast
+
+from linz_logger import get_log
+
+
+def format_source(source: List[str]) -> List[str]:
+    """Due to Argo constraints if using the basemaps cli list command
+    the source has a string that contains a list that needs to be split.
+    example: ["[\"s3://test/image_one.tiff\", \"s3://test/image_two.tiff\"]"]
+    """
+    if len(source) == 1 and source[0].startswith("["):
+        try:
+            return cast(List[str], json.loads(source[0]))
```

Comment:
Was `mypy` complaining so you had to add `cast`? which is actually not casting the Python object returned by `json.loads()` but used for `typing`. In our case the json is an array so `json.loads` returns a Python `list`